src/Symfony/Component/Console/ConsoleEvents.php

```diff
@@ -27,6 +27,14 @@ final class ConsoleEvents
      */
     const COMMAND = 'console.command';
 
+    /**
+     * The STOP event allows you to perform some actions
```

Comment:
```suggestion
     * The SIGNAL event allows you to perform some actions
```

Reply:
IMHO waiting the doc part, it can be good to add an usage in the PR description for example :)
with all signals etc
because it is a very usefull addition 👍

Reply:
Thank you!
I added to description an example of usage. Is it good for understanding?

Reply:
yes for me @marie 
what do you think adding a ref to the signals int value on the doc?
IMO such ref is good to have here on the code, and later on the doc :)